js.dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:10-slim
+
+RUN apt-get update && apt-get install -yq libgconf-2-4 \
+    && apt-get -y install sudo \
+    && apt-get install apt-transport-https -y
+
+# Install latest chrome dev package and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium Taiko installs, work.
+# Reference https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md
+RUN apt-get update && apt-get install -y wget --no-install-recommends \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get purge --auto-remove -y curl \
+    && rm -rf /src/*.deb
+
+# It's a good idea to use dumb-init to help prevent zombie chrome processes.
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
```

Comment:
I don't see an `ENTRYPOINT` corresponding to `dumb-init` - am I missing something? I am referring to [`dumb-init` usage](https://github.com/Yelp/dumb-init/#usage)